### PR TITLE
archive the run after transferring

### PR DESCRIPTION
**Pull request by Galithil**
Also, ensure that the samplesheet has the right sampleIDs

### CODE REVIEW

**Comment by guillermo-carrasco** [comment]
agh... I implemented this in my not-yet-submitted pull request. I'm gonna have so many merging conflicts :-(

File: taca/analysis/analysis.py [hunk 1]
```diff
@@ -91,9 +91,40 @@ def transfer_run(run, analysis=True):
...
+        logger.warn("Cannot find the runParameters.xml file at {}. This is quite unexpected. please archive the run {} manually".format(rppath, run))
+    else:
+        try:
+            #Works for recent control software
+            runtype=rp.data.["Setup"].get("Application Name")
+        except KeyError :
+            #should work for ancient control software
+            runtype=rp.data.get("Application Name")
+
+        if "HiSeq X" in runtype:
```
Comment (guillermo-carrasco) [inline taca/analysis/analysis.py]: hmmm you're assuming here an implicit order of `archive_dir` in the configuration paths, which I wouldn't do... these config files are changed quite often. What I did is try to find the run in one of the `data_dirs` and the append the `nosync` to the one that has the run.

Comment (Galithil) [inline taca/analysis/analysis.py]: that kinda defeats the point of having them in the config file. We could
change that to a key/value pair :
HiSeq : <path>
MiSeq : <path>
HiseqX: <path>

what do you think ?


On Tue, Jun 9, 2015 at 5:04 PM, Guillermo Carrasco <notifications@github.com

Comment (guillermo-carrasco) [inline taca/analysis/analysis.py]: that would be better yes, and it's actually what I did for `bcl2fastq`

```
bcl2fastq:
        HiSeqX: /home/hiseq.bioinfo/opt/bcl2fastq/bin/bcl2fastq
        HiSeq: /usr/local/bin/configureBclToFastq.pl
        MiSeq: /usr/local/bin/configureBclToFastq.pl
```

**Comment by Galithil** [comment]
so travis fails, but it fails on flowcell_parser, which now passes travis. How can I tell him to try again. Empty commit ? :x

**Pull request merged**